includes/pages/admin_active.php

```diff
@@ -179,18 +210,21 @@ function admin_active()
                 continue;
             }
         }
-        $usr['nick'] = User_Nick_render($usr);
-        $usr['shirt_size'] = $tshirt_sizes[$usr['Size']];
-        $usr['work_time'] = round($usr['shift_length'] / 60)
+
+        $userData = [];
+        $userData['nick'] = User_Nick_render($usr);
+        $userData['shirt_size'] = $tshirt_sizes[$usr->personalData->shirt_size];
```

Comment:
This fails when shirt_size is empty.
Add '' => '-' as shirt size to config.default.php.

Reply:
I really want keep the config file clean -> any "placeholder" stuff should be handelt inside the view.

I think that something like
```php
$userData['shirt_size'] = (isset($tshirt_sizes[$shirtSize]) ? $tshirt_sizes[$shirtSize] : '-');
```
would be better